spec/support/matchers/have_h1.rb

```diff
@@ -0,0 +1,5 @@
+RSpec::Matchers.define :have_h1 do |text|
+  match do |page|
+    Capybara.string(page.body).has_selector?("h1", text: text)
+  end
+end
```

Comment:
I feel like this is a bit of unnecessary abstraction - it only saves writing the word "selector" and won't be useful if need to use other options in "has_selector" method.

Reply:
I was in two minds about this one, so happy to remove it.